src/main/java/it/gov/pagopa/receipt/pdf/datastore/utils/BizEventToReceiptUtils.java

```diff
@@ -208,6 +208,13 @@ public static BigDecimal getAmount(BizEvent bizEvent) {
         return BigDecimal.ZERO;
     }
 
+    public static BigDecimal getCartAmount(BizEvent bizEvent) {
+        if (bizEvent.getTransactionDetails() != null && bizEvent.getTransactionDetails().getTransaction() != null) {
+            return formatEuroCentAmount(bizEvent.getTransactionDetails().getTransaction().getGrandTotal());
+        }
+        return BigDecimal.ZERO;
```

Comment:
I think that `BigDecimal.ZERO` as default can hide some problem with the biz event.
Throw an exception instead.